api/src/pcapi/core/highlights/models.py

```diff
@@ -48,8 +58,8 @@ class Highlight(PcObject, Model):
 
     @property
     def is_available(self) -> bool:
-        now = get_naive_utc_now()
-        return self.availability_timespan.upper >= now and self.availability_timespan.lower < now
+        today = datetime.date.today()
+        return self.availability_datespan.upper > today and self.availability_datespan.lower <= today
```

Comment:
question : pourquoi avoir inversé les condition? 

ie. pourquoi on n'a pas 

```suggestion
        return self.availability_datespan.upper >= today and self.availability_datespan.lower < today
```

Reply:
C'est effectivement fait exprès :)
La différence c'est qu'on gérait des datetime, maintenant on gère des dates.

On n'a moins de marge de manœuvre avec les date qu'avec les datetimes : il y a très peu de chance de se trouver dans une situation où on est à un moment de la journée où `now == valeur_de_un_datetime`. Donc on aurait pu complètement enlever les `=` dans la version précédente presque. En tout cas ça n'était pas correcte tel que fait avant et ça tombait en marche (cf le peu de probabilité de se retrouver avec `now == valeur_de_un_datetime`)

Par contre, avec des dates, ça devient critique. Il y a plus de moment de la journée où `today == valeur_de_une_date`.

En l’occurrence, une valo est stockée avec une limite inférieur inclusive et une limite supérieur exclusive. Donc pour dire dipo le 11/11/2025 on va avoir `self.availability_datespan.lower == 11/11/2025` mais `self.availability_datespan.upper == 12/11/2025`. Il faut que `self.availability_datespan.upper` soit strict dans sa comparaison à `today` (je sais pas si ça fait sens ce que je dis)

Reply:
c'est très clair merci